### ISSUE TITLE
TypeError: 'Buffer' does not support the buffer interface

### ISSUE THREAD

**Issue by scottbelden** [milesgranger/cramjam#116]
When using pypy 3.8 I get the error mentioned in the title. This doesn't seem to happen on pypy 3.9. Do you know what might be going on?

To reproduce, save the following contents as `compress.py`:
```python
from io import BytesIO
from cramjam import snappy

compressed = snappy.compress_raw(b"123")
BytesIO(compressed)
```

And then you can use the following `Dockerfile`:
```
FROM pypy:3.8-bullseye

RUN pip install cramjam
COPY compress.py /

RUN pypy compress.py
```

Note: Change the dockerfile to use `pypy:3.9-bullseye` to see that it works in 3.9

**Comment by milesgranger** [comment]
Thanks for the reproducer! 
I'll try to have a look this weekend.

**Comment by milesgranger** [comment]
Regarding this and the related issue, not ignoring.. just busy. :sweat_smile:

**Comment by scottbelden** [comment]
No worries. I completely understand. And I worked around it by just using 3.9.

**Comment by milesgranger** [comment]
Came to my attention that pypy3.8 isn't developed any longer and now w/ #144 in, it tests pypy3.9 and 3.10 so this ought to be fixed now in [v2.8.2](https://github.com/milesgranger/cramjam/releases/tag/v2.8.2) for the supported pypy versions.
